lib/salus/scanners/slither.rb

```diff
@@ -82,6 +82,8 @@ def config_opts
         #       A list of paths must be joined by | as in a regular expression, not comma
         opts += ' --filter-paths ' + @config['filter-paths'].to_s + ''
       end
+      opts += ' --exclude-informational' if @config.dig('exclude-informational')
```

Comment:
If we see a need for additional options I would be a fan of moving this to a dynamic OPT list.  I'm okay w/the static listing given we just have  2 bool and 1 string opt but any more would be nice to DRY up.